src/java.base/share/native/libjli/java.c

```diff
@@ -1518,6 +1518,10 @@ ParseArguments(int *pargc, char ***pargv,
         if (JLI_StrCmp(arg, "-Xshare:dump") == 0) {
             dumpSharedSpaces = JNI_TRUE;
         }
+        if (JLI_StrCmp(arg, "-XX:AOTMode=create") == 0) {
```

Comment:
This is inappropriate - the launcher does not, and should not, process hotspot -XX options. Any aliasing should happen in the hotspot argument processing logic.

Reply:
I realize this poses a problem with communicating to the launcher that this is a "terminal" flag. Maybe AOT should have -X flags instead of -XX?

Reply:
Hi David, thanks for the review. After off-line discussion, I think it's best to remove this check from the launcher, and exit the JVM directly when `-XX:AOTMode=create` is specified. This is the same way as how `-Xlog:help` is handled. See 

https://github.com/openjdk/jdk/blob/b711c41d442fc369a84745c0203db638e0b7e671/src/hotspot/share/runtime/arguments.cpp#L2588-L2591